dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -686,6 +686,13 @@ const textRichLink: (format: ArticleFormat) => string = (format) => {
 	return pillarPalette[ArticlePillar.News][400];
 };
 
+const hoverLiveblog = (format: ArticleFormat): string => {
+	if (format) {
```

Comment:
Did you mean to check a property of `format` here?

Reply:
I assumed this would need to be checked in order to return the right colour for different types of blog. I can take it out if you want.